Global/Linux.gitignore

```diff
@@ -11,3 +11,9 @@
 
 # .nfs files are created when an open file is removed but is still being accessed
 .nfs*
+
+# Log files created by default by the nohup command
+nohup.out
+
+# Metadata left by Dolphin file manager, which comes with KDE Plasma
+.directory
```

Comment:
This is actually already included on [lines 6-7 above](https://github.com/github/gitignore/pull/4489/files#diff-8704ae78f5e916819383a0c34fab3c7d7acdc2b6fc15dd189184780821f734edR6-R7)!